### PR TITLE
Minor fixes and kwc-blockly bump

### PR DESCRIPTION
**Pull request by pazdera**

### CODE REVIEW

File: src/app/elements/kc-workspace-frame/kc-parts-controls.ts [hunk 1]
```diff
@@ -239,6 +240,9 @@ export class KCPartsControls extends LitElement {
         if (existing) {
             return 'A part with that name already exist';
         }
+        if (!newName.length) {
+            return 'A part name can\'t be empty';
```
Comment (rachaelcodes) [inline src/app/elements/kc-workspace-frame/kc-parts-controls.ts]: could we add the i18n to this file?

**Comment by TeamKano** [comment]
<table>
    <tr>
        <th colspan="3">Coverage summary</th>
    </tr>
<tr>
        <td>Statements</td>
        <td>29.35%</td>
        <td>2002/6820</td>
    </tr>
<tr>
        <td>Branches</td>
        <td>20.21%</td>
        <td>209/1034</td>
    </tr>
<tr>
        <td>Functions</td>
        <td>29.35%</td>
        <td>648/2208</td>
    </tr>
<tr>
        <td>Lines</td>
        <td>29.24%</td>
        <td>1935/6618</td>
    </tr>
</table>

*From Jenkins: kano-code/PR-1814*

File: src/app/elements/kc-workspace-frame/kc-parts-controls.ts (hunk 1 above)
Comment (pazdera) [reply src/app/elements/kc-workspace-frame/kc-parts-controls.ts]: Yes! I forgot about that. I'll do that now.

Comment (pazdera) [reply src/app/elements/kc-workspace-frame/kc-parts-controls.ts]: @rachaelcodes The validation errors should be translated now.

**Pull request merged**